test/Rollback.integration.t.sol

```diff
@@ -64,14 +65,13 @@ contract ProposeWithRollback is RollbackIntegrationTest {
     assertEq(fakeProtocolContract.fee(), feeWhenProposalIsExecuted);
     assertEq(fakeProtocolContract.feeGuardian(), feeGuardianWhenProposalIsExecuted);
 
-    // 4. Verify rollback is proposed to URM
+    // 4. Verify rollback is proposed to URM and is in pending state
     uint256 _rollbackId = rollbackHelper.getRollbackId(feeWhenRollbackIsExecuted, feeGuardianWhenRollbackIsExecuted);
-    assertTrue(upgradeRegressionManager.isRollbackEligibleToQueue(_rollbackId));
+    assertEq(uint8(upgradeRegressionManager.state(_rollbackId)), uint8(ProposalState.Pending));
```

Comment:
It might be worth creating a custom `_assertEqState` helper method to avoid all this ugly casting everywhere